### DISCUSSION TITLE
How to get linked issue from pull_request response of api

### DISCUSSION THREAD

**Discussion by Yaminyam** [community/community#30365] [Apps, API and Webhooks] [answered]
I tried to develop an action app using github rest api.
pr on github has a feature called Development.
However, in the pull_request api response, I could not find any information about the issue related to that development.
Is there a way to get information on issues linked from pr to development?

**Comment by Yaminyam** [comment, marked as answer]
I'm leaving this for anyone who may have the same problem as me in the future.
There are differences in GitHub API between rest and graphql.
Most of the latest features such as development exist in the graphql API.

**Comment by Reemoradwan** [comment]
سأترك هذا لأي شخص قد يواجه نفس مشكلتي في المستقبل.
هناك اختلافات في GitHub API بين الراحة وgraphql.
معظم أحدث الميزات مثل التطوير موجودة في واجهة برمجة تطبيقات graphql.
